src/main/java/nl/rijksoverheid/mev/gezagsmodule/domain/ARVragenModel.java

```diff
@@ -80,8 +81,8 @@ public class ARVragenModel {
     private final GezagService gezagService;
 
     public ARVragenModel(final Persoonslijst plPersoon,
-            final GezagService gezagService,
-            final Transaction transaction) {
+                         final GezagService gezagService,
+                         final Transaction transaction) {
```

Comment:
Rare formatting? Heel veel spaces

Reply:
is default intelij, uitlijning met parameter erboven